src/js/tooltip.js

```diff
@@ -363,16 +363,13 @@ class Tooltip {
 
 		if (this.opts.appendToBody) {
 			// If the tooltip will be apended directly to body:
-			// use absolute coordinates of the target relative to its position inside the page
```

Comment:
By adding the tooltip to the `body` we have to set the coordinates of the tooltip relative to the position of the targeted element inside the `body` and not relative to the direct parent of the targeted element.
So by doing this

```js
this.tooltipEl.style.top = (this.tooltipRect.top - targetTopOffset) + 'px';
this.tooltipEl.style.left = (this.tooltipRect.left - targetLeftOffset) + 'px';
```

instead of 

```js
this.tooltipEl.style.top = this.tooltipRect.top + 'px';
this.tooltipEl.style.left = this.tooltipRect.left + 'px';
```

will result in:

![image](https://user-images.githubusercontent.com/15702167/43456394-feeba2e6-94cb-11e8-9c15-57c37689b5e4.png)

*In this case the targeted element is the marked search input*

Reply:
Ah right you are! The reason the demo I added wasn't working is because page scroll wasn't accounted for, which doesn't make a difference to your usecase. I've updated that!